packages/docs/docs/guides/5-request-client/7-encryption.md

```diff
@@ -4,13 +4,9 @@ keywords: [Request, new currency, test network, missing currency, testnet]
 description: Learn how to integrate Request network and its features.
 ---
 
-| WARNING: The Request encryption mechanism is still under audit, use it at your own risk! |
-| ---------------------------------------------------------------------------------------- |
-
-
-| WARNING: Manipulate private keys must be done with care, losing them can lead to a loss of data and privacy! |
-| ------------------------------------------------------------------------------------------------------------ |
-
+:::caution
+Manipulate private keys must be done with care, losing them can lead to a loss of data and privacy!
```

Comment:
**Manipulating** private keys must be done with care, losing them can lead to a loss of data and privacy!